.github/workflows/build.yml

```diff
@@ -1,9 +1,7 @@
 name: build
 on:
-  push:
-    branches: [master]
-  pull_request:
-    branches: [master]
+  - push
```

Comment:
Temporary changes while I test the github build action against my forked branch.